Setup/UI/TelemetryDlg.wxs

```diff
@@ -14,11 +14,10 @@
       <![CDATA[Installed OR POWERSHELLEXE]]>
     </Condition>
 
-    <!-- Windows 7SP1 supports only PowerShell 2.0 -->
```

Comment:
Why remove this comment, even if not forcing v2.0?

Reply:
It was kind of misleading. Win7 SP1 supports _at maximum_ PowerShell 2.0, but there's no guarantee that it is actually available on all Win7 systems. It depends on the installed .NET Framework version.